packages/core/src/lib/filter/index.ts

```diff
@@ -228,6 +227,7 @@ class Subscription {
 class Filter extends BaseProtocol implements IReceiver {
   private readonly options: ProtocolCreateOptions;
   private activeSubscriptions = new Map<string, Subscription>();
+  private readonly NUM_PEERS_PROTOCOL = 1;
```

Comment:
I see that we always define `constants` in a file scope, like `FilterCodecs` up there.

Reply:
Maybe we can place in `core/constants` or something?

Reply:
no problems with codecs, we have other places with constants defined on top
https://github.com/waku-org/js-waku/blob/9fc79f6853ac5dd7a8411fff57a0d23f3b9dfe77/packages/core/src/lib/connection_manager.ts#L21

Reply:
oh, I don't really think it should be a separate package, to me it seems good enough that we have `constants.ts` files